docs/integrations/transformers.md

```diff
@@ -77,16 +77,30 @@ cloud of your choice (AWS, GCP, Azure) and automatically deploy ClearML agents:
 and shuts down instances as needed, according to a resource budget that you set.
 
 
-### Cloning, Editing, and Enqueuing
+### Reproducing Tasks
 
 ![Cloning, editing, enqueuing gif](../img/gif/integrations_yolov5.gif#light-mode-only)
 ![Cloning, editing, enqueuing gif](../img/gif/integrations_yolov5_dark.gif#dark-mode-only)
 
-Use ClearML's web interface to edit task details, like configuration parameters or input models, then execute the task 
-with the new configuration on a remote machine:
-* Clone the task
-* Edit the hyperparameters and/or other details 
-* Enqueue the task
+Use ClearML's web interface to reproduce tasks and edit their details, like hyperparameters or input models, then execute the tasks 
+with the new configuration on a remote machine.
+
+When ClearML is integrated into a script, it captures and stores configurations, such as hyperparameters 
+and model settings. Typically, when a ClearML Agent re-executes a task, the agent users the values stored in the task,
```

Comment:
@pollfly "Typically...hard-coded values" -> "When executing a task, the ClearML Agent will, by default, override runtime configuration values (such as hyperparameters and environment variables) with the values specified in the task."